infrastructure/azure.tf

```diff
@@ -713,37 +808,37 @@ resource "null_resource" "azurerm_function_app_git" {
   }
 }
 
-locals {
-  application_outbound_ips = "${var.azurerm_shared_address_space_cidr},${azurerm_function_app.azurerm_function_app.outbound_ip_addresses},${var.azurerm_azure_portal_ips}"
-}
-
-resource "null_resource" "azurerm_cosmosdb_ip_range_filter" {
-  triggers = {
-    cosmosdb_id              = "${azurerm_cosmosdb_account.azurerm_cosmosdb.id}"
-    application_outbound_ips = "${local.application_outbound_ips}"
-
-    # increment the following value when changing the provisioner script to
-    # trigger the re-execution of the script
-    # TODO: consider using the hash of the script content instead
-    provisioner_version = "1"
-  }
-
-  provisioner "local-exec" {
-    command = "${join(" ", list(
-      "ts-node ${var.cosmosdb_iprange_provisioner}",
-      "--resource-group-name ${azurerm_resource_group.azurerm_resource_group.name}",
-      "--cosmosdb-name ${azurerm_cosmosdb_account.azurerm_cosmosdb.name}",
-      "--ips ${local.application_outbound_ips}"))
-    }"
+# locals {
```

Comment:
can we just remove the commented lines? we still have them in the git history

Reply:
I'm totally with you. I'd just prefer to do it in the next cleanup patches.